applications/imaging_ai_segmentator/README.md

```diff
@@ -1,198 +1,157 @@
+3. **Download test data (if not already done):**
+   - Download CT series from [TCIA](https://nbia.cancerimagingarchive.net/nbia-search/)
+   - Save DICOM files under `$HOLOSCAN_INPUT_PATH`
 
-Set the `PYTHONPATH` to include the Holohub source folder
-```bash
-export PYTHONPATH=$PYTHONPATH:$PWD
-```
+4. **Install dependencies:**
 
-Set the environment variables for the application
-```bash
-source applications/imaging_ai_segmentator/env_settings.sh
-```
+   ```bash
+   pip install -r applications/imaging_ai_segmentator/requirements.txt
+   ```
 
-If not already done, download images of a CT series from [TCIA](https://nbia.cancerimagingarchive.net/nbia-search/),  unzip if necessary, and save the folder of DICOM files under the folder `$HOLOSCAN_INPUT_PATH`.
+5. **Build and install the application:**
 
-Optionally download the AI model from [MONAI Model Zoo](https://github.com/Project-MONAI/model-zoo/tree/dev/models/wholeBody_ct_segmentation), or wait till the build step to have it downloaded automatically
+   ```bash
+   ./run build imaging_ai_segmentator
+   ```
 
-```bash
-mkdir -p $HOLOSCAN_MODEL_PATH
-pip install gdown
-python -m gdown https://drive.google.com/uc?id=1PHpFWboimEXmMSe2vBra6T8SaCMC2SHT -O $HOLOSCAN_MODEL_PATH/model.pt
-```
+6. **Run the application:**
 
-Install Python packages required by the application
-```bash
-pip install -r applications/imaging_ai_segmentator/requirements.txt
-```
-Run the application
-```bash
-rm -f -r $HOLOSCAN_OUTPUT_PATH
-python applications/imaging_ai_segmentator/
-```
+   ```bash
+   rm -fr $HOLOSCAN_OUTPUT_PATH
+   python install/imaging_ai_segmentator/app.py
+   ```
 
-Check output
-```bash
-ls $HOLOSCAN_OUTPUT_PATH
-```
+   **_Tip:_**
+   You can override the default input, output, and model directories by specifying them as command-line arguments. For example:
 
-## Packaging the Application for Distribution
+   ```bash
+   python install/imaging_ai_segmentator/app.py -m /path/to/model -i /path/to/input -o /path/to/output
+   ```
 
-With Holoscan CLI, an applications built with Holoscan SDK can be packaged into a Holoscan Application Package (HAP), which is essentially a Open Container Initiative compliant container image. An HAP is well suited to be distributed for deployment on hosting platforms, be a Docker Compose, Kubernetes, or else. Please refer to [Packaging Holoscan Applications](https://docs.nvidia.com/holoscan/sdk-user-guide/holoscan_packager.html) in the User Guide for more information.
+7. **Check output:**
 
-This example application provides all the necessary contents for HAP packaging, and the specific commands are revealed by the specific commands.
+    ```bash
+    ls $HOLOSCAN_OUTPUT_PATH
+    ```
 
-**_Note_**
+## Output
 
-The prerequisite is that the application `build_and_install` has been performed to stage the source and AI model files for packaging.
+The application generates two types of outputs:
 
-```
-source applications/imaging_ai_segmentator/packageHAP.sh
```

Comment:
Is there a reason to remove the instruction to package the app into a Holoscan App Package?